### DISCUSSION TITLE
Can I "Run Interactive" with a volume attached?

### DISCUSSION THREAD

**Discussion by montmejat** [microsoft/vscode-docker#4332] [Q&A]
I want to start a container in interactive mode with a volume attached, exactly like I would do with: `docker run -it -v /home/user/Datasets/:/datasets ros:noetic bash`.

I see that's there an option to start a container in interactive mode:
![Screenshot from 2024-06-26 14-21-26](https://github.com/microsoft/vscode-docker/assets/29719028/87716532-aa52-45d3-a4dc-58ede17a1e96)

Just by curiosity, is there a way to start it with a volume attached? :)

**Comment by bwateratmsft** [comment]
Yes, you can leverage [command customization](https://code.visualstudio.com/docs/containers/reference#_command-customization) to define a custom command that includes a volume mount.
